site/profile/manifests/core/openvpnas.pp

```diff
@@ -0,0 +1,24 @@
+class profile::core::openvpnas {
```

Comment:
There are several openvpn modules on the forge.  Is there a reason why we can not use one of the existing modules?

Reply:
Hi @jhoblitt. The reason for this is that since it was my first puppet installation, I really wanted to learn how to automate this whole process using a more traditional approach with puppet.  This process helped me learn and understand class creations and roles more in-depth.